### ISSUE TITLE
Cancelling a reindex takes 2+ hours

### ISSUE THREAD

**Issue by danielsotirhos** [DataBiosphere/azul#3304]
Attempting to cancel an ongoing reindex with `python scripts/manage_queues.py purge_all` can take over 2 hours to complete

The attached log shows:
```
13:09:54: Reindex started 
13:21:10: Reindex command cancelled (Ctrl-C)
13:21:27: "purge_all" action started
15:40:47: "purge_all" completed
```
[reindex-log.txt](https://github.com/DataBiosphere/azul/files/6942044/reindex-log.txt)

**Comment by hannes-ucsc** [comment]
Spike to check if the script can be dislodged by purging the queues with in-flight message while it is stuck on 

```
2021-08-05 13:42:48,630 INFO    ThreadPoolExecutor-0_5: Queue 'https://queue.amazonaws.com/122796619775/azul-notifications_retry-daniel' has 29 in-flight messages
```

**Comment by danielsotirhos** [comment]
> Spike to check if the script can be dislodged by purging the queues with in-flight message while it is stuck on

Yes, the following log file show an attempt to `manage_queues.py purge_all` during a reindex, and then being able to break out of the wait for in-flight messages by activating a purge action on the queue using the AWS console.
[purge-queue.txt](https://github.com/DataBiosphere/azul/files/7172311/purge-queue.txt)

**Comment by achave11** [comment]
Assignee to add code that purges all queues using the SQS API in a place that emulates what @danielsotirhos did above.

**Comment by hannes-ucsc** [comment]
I don't think purging the queues is the right thing to do. What's going on here is that setting reserved concurrency to 0 does not prevent the SQS trigger from pulling messages off the queue and at least attempting to invoke the function. So messages will continue to cycle between queued and in-flight state. Instead of setting the concurrency to 0 we will need to disable the trigger. Or maybe both.

**Comment by hannes-ucsc** [comment]
It is a regression from that commit. Need fix pronto, hence assigning to me.
